src/components/player/cue-bar.tsx

```diff
@@ -161,7 +161,14 @@ const NoteCue: React.FC<any> = ({
   // added seeking to the list here but getting some janky perf issues
 
   const startPosition = `${(cue.startTime / duration) * 100}%`
-  const note = cue.text
+
+  let note: {text: string; type?: string}
+
+  try {
+    note = JSON.parse(cue.text)
```

Comment:
I'm gonna try to parse it as JSON first to get a "note object" otherwise I assume it's a string and there was an error